xdocs/usermanual/jmeter_tutorial.xml

```diff
@@ -110,12 +110,28 @@ The folders inside of <code>src</code>
 </p>
 
 <dl>
+<!--
+  <dt><code>bom</code></dt><dd></dd>
```

Comment:
This list would easily go out of date :-/
Does it mean every module we add should be updated here?

Reply:
I think it 's useful.

Reply:
I think we should approach `bom` in a different way: we should probably add a section on "how to add a new third-party dependency to JMeter" and "how to update dependency version".